src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.html

```diff
@@ -1,116 +1,121 @@
+            <div>
+                {{ 'artemisApp.courseOverview.exerciseDetails.assessmentType' | artemisTranslate }}
             </div>
-            <div class="row" *ngIf="exercise.difficulty">
-                <div class="col-6">
-                    <h5>{{ 'artemisApp.courseOverview.exerciseDetails.difficulty' | artemisTranslate }}</h5>
-                </div>
-                <div class="col-6">
-                    <h5>
-                        <jhi-difficulty-badge [exercise]="exercise"></jhi-difficulty-badge>
-                    </h5>
-                </div>
+            <div>
+                {{ 'artemisApp.AssessmentType.forExerciseHeader.' + exercise.assessmentType | artemisTranslate }}
+                <fa-icon [icon]="faQuestionCircle" ngbTooltip="{{ 'artemisApp.AssessmentType.tooltip.' + exercise.assessmentType | artemisTranslate }}"></fa-icon>
             </div>
-            <div class="row" *ngIf="exercise.maxPoints != undefined">
-                <div class="col-6">
-                    <h5>{{ 'artemisApp.courseOverview.exerciseDetails.points' | artemisTranslate }}</h5>
-                </div>
-                <div class="col-6">
-                    <h5>
-                        {{ exercise.maxPoints }}
-                        <span *ngIf="exercise.bonusPoints">({{ 'artemisApp.courseOverview.exerciseDetails.bonus' | artemisTranslate }} {{ exercise.bonusPoints }})</span>
-                    </h5>
-                </div>
+        </div>
+        <div class="submission-row" *ngIf="submissionPolicy">
+            <div>{{ 'artemisApp.programmingExercise.submissionPolicy.submissionLimitTitle' | artemisTranslate }}:</div>
+            <div>
+                {{
+                    numberOfSubmissions +
+                        '/' +
+                        submissionPolicy.submissionLimit +
+                        (submissionPolicy.exceedingPenalty
+                            ? ('artemisApp.programmingExercise.submissionPolicy.submissionPenalty.penaltyInfoLabel'
+                              | artemisTranslate: { points: submissionPolicy.exceedingPenalty })
+                            : '')
+                }}
+                <fa-icon
+                    [icon]="faQuestionCircle"
+                    ngbTooltip="{{ 'artemisApp.programmingExercise.submissionPolicy.submissionPolicyType.' + submissionPolicy.type + '.tooltip' | artemisTranslate }}"
+                ></fa-icon>
             </div>
-            <div class="row" *ngIf="submissionPolicy != undefined">
-                <div class="col-6">
-                    <h5>{{ 'artemisApp.programmingExercise.submissionPolicy.submissionLimitTitle' | artemisTranslate }}:</h5>
+        </div>
+    </div>
+    <div class="right-col">
+        <ng-template [ngIf]="exam" [ngIfElse]="courseMode">
+            <ng-container *ngIf="!isTestRun">
+                <div [ngClass]="{ 'fw-500': isNextDueDate[0] }">
```

Comment:
```suggestion
                <div [class.fw-500]="isNextDueDate[0]">
```
Should also work (multiple times in you code)